motion/thruster_interface_auv/package.xml

```diff
@@ -7,13 +7,12 @@
   <maintainer email="vortex.git@vortexntnu.no">vortex</maintainer>
   <license>MIT</license>
 
-  <buildtool_depend>ament_cmake_python</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <depend>rclpy</depend>
+  <depend>rclcpp</depend>
   <depend>std_msgs</depend>
   <depend>vortex_msgs</depend>
-
-  <exec_depend>ros2launch</exec_depend>
+  <depend>ament_index_cpp</depend>
```

Comment:
why do you need to depend on ament_index_cpp?

Reply:
gpt-suggested stuff there from the beginning. Imma remove it :)